bulktest/batch_lib.inc

```diff
@@ -661,9 +661,8 @@ function importEntries($entries, $pageid, $statusInfo, &$firstUrl, &$firstHtmlUr
 			$bFirstReq = 1;
 			$firstUrl = $url;
 		}
-		if ( ! $firstHtmlUrl && 200 == $status ) {
+		if ( ! $firstHtmlUrl && 200 == $status && $type == 'html' ) {
 			// This is the first URL found associated with the page that's HTML.
-			// We could check that mimeType like "%html%", but some HTML responses have the wrong content-type.
```

Comment:
"but some HTML responses have the wrong content-type"... any chance we can quantify if/how big of a problem this is? Naively.. find pages that have <body> tag but non html content type, or some such?

Also, add a comment to note why we're filtering by HTML?